test/helpers/sign.js

```diff
@@ -9,6 +9,15 @@ function toEthSignedMessageHash (messageHex) {
   return web3.utils.sha3(Buffer.concat([prefix, messageBuffer]));
 }
 
+function fixSignature (signature) {
+  //in geth its always 27/28, in ganache its 0/1. Change to 27/28 to prevent
+  //signature malleability
+  //https://github.com/ethereum/go-ethereum/blob/master/internal/ethapi/api.go#L447
+  const v = parseInt(signature.slice(130, 132), 16) + 27;
```

Comment:
We may want to add a check here so that the 27 is only added if `v` is 0 or 1, to prevent `fixSignature` from accidentally breaking correct signatures.

Reply:
agreed